io/plugins/eu.esdihumboldt.hale.io.csv/src/eu/esdihumboldt/hale/io/csv/reader/internal/CSVConfiguration.java

```diff
@@ -36,14 +36,14 @@
 
 	private final char escape;
 
-	private final boolean skip_first_line;
+	private final int skip_N_lines;
```

Comment:
Java naming standard with `camelCase`. Underscore is only allowed for constants and in-between `ALL_CAPITAL_LETTERS`.